frontend/public/components/operator-lifecycle-manager/create-operand.tsx

```diff
@@ -141,8 +140,20 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
         <Alert isInline className="co-alert co-break-word" variant="danger" title={error} />
       </div>}
       <div className="row">
-        <button className="btn btn-primary" type="submit" onClick={submit}>Create</button>
-        <Link className="btn btn-default" to={`/k8s/ns/${props.namespace}/clusterserviceversions/${props.clusterServiceVersion.metadata.name}/${referenceForModel(props.operandModel)}`}>Cancel</Link>
+        <ActionGroup className="pf-c-form">
+          <Button
+            onClick={submit}
+            type="submit"
+            variant="primary">
+            Create
+          </Button>
+          <Button
+            component="a"
+            href={`/k8s/ns/${props.namespace}/clusterserviceversions/${props.clusterServiceVersion.metadata.name}/${referenceForModel(props.operandModel)}`} id="cancel"
```

Comment:
`id="cancel"` is new.  Do you need it?  If so, it should go on its own line.

Reply:
> `id="cancel"` is new. Do you need it? If so, it should go on its own line.

I'd rather not add `id` unless necessary (e.g. for associating labels with inputs). If we need it for tests, we should prefer `data-test-id` attributes.